CMakeLists.txt

```diff
@@ -249,6 +250,24 @@ if(PCAPPP_TARGET_COMPILER_CLANG
   add_compile_options(-Wall)
 endif()
 
+if(PCAPPP_BUILD_REPRODUCIBLE)
+  add_definitions(-DPCAPPP_BUILD_REPRODUCIBLE)
+  if(APPLE)
+    if(NOT $ENV{ZERO_AR_DATE})
+      message(FATAL_ERROR "You need to set `export ZERO_AR_DATE=1`")
+    endif()
+  elseif(MSVC)
+    add_compile_options(/Brepro)
+    add_compile_options(/experimental:deterministic)
+    add_link_options(/Brepro)
+    add_link_options(/experimental:deterministic)
+    add_link_options(/INCREMENTAL:NO)
```

Comment:
If we assume MSVC doesn't work with reproducible build, do we need these special options?

Reply:
I doesn't work for static Library but should work for shared or executable

Reply:
shared doesn't work on Windows: https://github.com/seladb/PcapPlusPlus/issues/1396

I can test executables, but not sure it's worth the effort, maybe we should just remove it?

Reply:
I will change it to a comment and mark it has unsupported for the moment